src/atoms/Icon/index.js

```diff
@@ -31,7 +33,11 @@ Icon.propTypes = {
   /**
    * The icon you'd like to display.
    */
-  icon: PropTypes.string
+  icon: PropTypes.string,
+  /**
+   * click handler for the wrapper div around the svg
```

Comment:
I know we have yet to standardize this, but should write these with the first letter capitalized?